Source/Scene/UrlTemplateImageryProvider.js

```diff
@@ -274,6 +244,11 @@ define([
          */
         tileWidth : {
             get : function() {
+                //>>includeStart('debug', pragmas.debug);
+                if (!this.ready) {
+                    throw new DeveloperError('tileWidth must not be called before the imagery provider is ready.');
```

Comment:
Is there a way I can artificially make the imageryProvider return `ready = false` in order to test the `DeveloperErrors`?